NGitLab.Tests/PackageTests.cs

```diff
@@ -0,0 +1,40 @@
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using NGitLab.Models;
+using NGitLab.Tests.Docker;
+using NUnit.Framework;
+
+namespace NGitLab.Tests
+{
+    public class PackageTests
+    {
+        [Test]
+        [NGitLabRetry]
+        public async Task Test_publish_package()
+        {
+            using var context = await GitLabTestContext.CreateAsync();
+            var project = context.CreateProject();
+            var packagesClient = context.Client.Packages;
+
+            var packagePublish = new PackagePublish
+            {
+                FileName = "README.md",
+                PackageName = "Packages",
+                PackageVersion = "1.0.0",
+                Status = "default",
+                PackageStream = File.OpenRead("../../../../README.md"),
```

Comment:
You'll need to dispose the stream. Also, don't rely on the file system, you can use a `MemoryStream` to create a dummy content.